lib/minitar/pax_header.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+class Minitar
+  # Implements the PAX Extended Header as a Ruby class. The header consists of following strings:
+  #
+  #    "#{length} #{keyword}=#{value}\n"
```

Comment:
What's the encoding of `length` here? I also think that I'd prefer seeing this written as:

```suggestion
  #    <wordlength><space><asciikeyword>=<asciivalue>\n
```

Assuming that is correct.